infrastructure/lib/addons/perfdash.ts

```diff
@@ -0,0 +1,241 @@
+import { Construct } from 'constructs';
+import { Aws, CfnStack, StackProps } from 'aws-cdk-lib';
+import { aws_eks as eks } from 'aws-cdk-lib';
+import { aws_iam as iam } from 'aws-cdk-lib';
+
+export interface PerfDashProps extends StackProps {
+  cluster: eks.Cluster;
+  namespace: string;
+}
+
+export class PerfDash extends Construct {
+  constructor(scope: Construct, id: string, props: PerfDashProps) {
+    super(scope, id);
+
+    const ns = props.cluster.addManifest('perfdash-namespace', {
+      apiVersion: 'v1',
+      kind: 'Namespace',
+      metadata: {
+          name: props.namespace
+      }
+  })
+
+    const sa = props.cluster.addServiceAccount('perfdash-sa', {
+      name: 'perfdash-log-fetcher',
+      namespace: props.namespace
+    });
+
+    sa.node.addDependency(ns)
+    sa.role.attachInlinePolicy(new iam.Policy(this, 'perfdash-policy', {
+      statements: [
+        new iam.PolicyStatement({
+            resources: ['*'],
+            actions: [
+              // S3 readonly access
+              "s3:Get*",
+              "s3:List*",
+              "s3-object-lambda:Get*",
+              "s3-object-lambda:List*",
+            ],
+        }),
+      ],
+  }));
+
+  const cm = props.cluster.addManifest('perfdash-cm', {
+    apiVersion: 'v1',
+    kind: 'ConfigMap',
+    metadata: {
+        name: "perfdash-config",
+        namespace: props.namespace,
+        labels: {
+            "app.kubernetes.io/name": "perfdash",
+            "app.kubernetes.io/component": "config",
+            "app.kubernetes.io/part-of": "perfdash",
+        },
+    },
+    data: {
+        "PERFDASH_LOG_BUCKET": "my-bucket-name",
```

Comment:
Having this name like this is not going to be helpful for any customer unless we take this as an input param as part of CDK args otherwise when this config is applied by CDK this won't have any real value IIUC. Thoughts ?

Reply:
Responded in https://github.com/awslabs/kubernetes-iteration-toolkit/pull/400#discussion_r1179549785.
Even if we let user specify log bucket in flag, it'd be hard to do the same thing for `jobs.yaml`. A user managed configmap out of CDK would make more sense.

Reply:
Yup, that comment of mine came later after this comment when i was reviewing. Let's remove this out of the cdk part.